File: abc/abc176/b.py
# https://atcoder.jp/contests/abc176/submissions/38128758

s = list(input())

ans = 0
for c in s:
  x = ord(c) - 48
  ans += x
  # print(x, ans)

print("Yes" if ans % 9 == 0 else "No")

File: abc/abc176/b.cpp
// https://atcoder.jp/contests/abc176/submissions/27276358

#include <bits/stdc++.h>
using namespace std;
int main() {
  int ans = 0;
  char c;
  while(cin >> c) ans += (int)c - 48;
  if(ans % 9 == 0) cout << "Yes";
  else cout << "No";
}
